### PR TITLE
Update number-extensions-part-1.md

### PR DESCRIPTION
**Pull request by nem035**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/javascript/ecmascript-2015/number-extensions/number-extensions-part-1.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/ecmascript-2015/number-extensions/number-extensions-part-1.md?ref=ce1a659b456d400eb63ae54354357b4219acfb94)

File: javascript/ecmascript-2015/number-extensions/number-extensions-part-1.md [hunk 1]
```diff
@@ -93,23 +104,15 @@ Complete the missing calls with methods such that all the statements are true:
...
-* NaN
-* isNotNaN
- 
----
-## Footnotes
+* false, true, false
+* false, false, false
+* true, true, false
+* false, true, true
+* true, false, true
```
Comment (kapnobatai137) [inline javascript/ecmascript-2015/number-extensions/number-extensions-part-1.md:117]: these answers might be a bit hard to choose, as you have to do a bit of searching between the answers to find the right one:

image

we could do a mix of the old question and the new one, something like:
``````
```javascript
Number.isNan('enki');
// ???
Number.isFinite(0);
// ???
Number.???(9.81);
// false
```

* false
* true
* isInteger
* isNaN
``````

what do you think @nem035 ?

Comment (nem035) [reply javascript/ecmascript-2015/number-extensions/number-extensions-part-1.md:117]: Ideally, we'll use value comparison here but that won't be ready for a bit longer.

I'm ok with changing it for now if you'd like, no strong preference either way

Comment (kapnobatai137) [reply javascript/ecmascript-2015/number-extensions/number-extensions-part-1.md:117]: hmmm, let's leave it as is for now and improve when we have value comparison :+1:

**Pull request merged**